### ISSUE TITLE
Feature request: Delete old packages

### ISSUE THREAD

**Issue by phantomic12** [pacman-repo-builder/pacman-repo-builder#60]
Project works really well, though as time goes on, old packages build up, which can greatly increase the workflow execution time and in the case of github actions, the runners can fill up if the repo gets too large.

I don't have any experience in Rust, but there are a few ways that it might be possible to implement:

Add an option that deletes all old packages except the one most recently built

If there is a need to keep previous versions of older packages:

Add a global or per package option that dictates how many old versions of a package should be kept, autodeleting the oldest package by date modified, as git versions are near impossible to sort afaik

**Comment by KSXGitHub** [comment]
Good idea. I have the same problem as you too. Unfortunately, I haven't find the time to implement this. If you can make a pull request, I'll review and merge. If you don't, that's fine, but you would have to wait (for a long time most likely).

**Comment by xzsk2** [comment]
I wrote a simple script to clean old packages as a temporary workaround, will only keep the newest version of packages

Check [package-cleanup.sh](https://github.com/xzsk2/PKGBUILDS/blob/main/package-cleanup.sh) and use case [here](https://github.com/xzsk2/PKGBUILDS/blob/7d6f33f39b71c7041b3a6f3606e91a3fab168b08/.github/workflows/build.yaml#L52). Hope this is helpful
